### DISCUSSION TITLE
Multiple rows of tasks

### DISCUSSION THREAD

**Discussion by SpaceCowboy55** [lawand-dot-io/taskbar#23] [Feature requests]
Please add an option to allow multiple rows.  In uBar, I can drag the height to show two rows.

![uBarAsTwoRows](https://github.com/lawand-dot-io/taskbar/assets/160819173/7d2e39c8-84d9-41d5-a4f3-90fbde2a7fc0)

**Comment by lawand-dot-io** [comment]
Interesting idea, for me personally when I do a lot of multi-tasking and have many windows open, I separate them into different Spaces, so I don't run into this issue. I will keep this idea in mind, thanks for the suggestion :)

**Reply by pingoklooi** [reply]
Please make this happen. The reason why people need a tool like yours is because they have too many items open. There is a real need for even triple or quadruple line taskbars.

**Reply by pingoklooi** [reply]
Btw, this is one of the best task bar tools I've tried. Good job. Thumbnails could be snappier to display upon rollover though.

**Reply by lawand-dot-io** [reply]
@pingoklooi In the latest version (1.2.1) I added an option to make thumbnails appear more quickly

**Comment by SpaceCowboy55** [comment]
Yeah – I’ve tried that when my taskbar gets really cluttered but it is actually more effort and less efficient for me.  I support several clients at a given time and the work involves flipping through many different, concurrent apps (Chrome, Firefox, BIRT, Excel, Visual Studio Code, Oxygen, Teams, Outlook)

Thanks,
Steve

**Comment by SpaceCowboy55** [comment]
Should also mention that it doesn’t need to be draggable.  Setting # of rows in the preferences would be just fine, too.

Thanks,
Steve

**Comment by semaj1919** [comment]
Just want to add, this would be a useful feature.  Spaces isn't enough for me personally, as I tend to have a lot of browser windows open (and they're all part of the same space).

**Comment by lawand-dot-io** [comment]
Ok, I understand. I will consider this feature once I finish higher priority features like multi-monitor support :)

**Comment by platyase** [comment]
I would love to see this feature too.  I use two or three rows in the Windows taskbar, and I was also using it with uBar, but uBar crashes frequently on my system.  Taskbar has been stable for me, but I'd love to have multiple rows.
